src/transformations.jl

```diff
@@ -8,16 +8,16 @@ abstract type AbstractDataTransform end
 
 Apply transformation `t` to vector or matrix `x` in place.
 """
-transform!(t::AbstractDataTransform, x::AbstractArray{<:Real,1}) = transform!(x, t, x)
-transform!(t::AbstractDataTransform, x::AbstractArray{<:Real,2}) = transform!(x, t, x)
+transform!(t::AbstractDataTransform, x::AbstractMatrix{<:Real}; dims::Int=1) = transform!(x, t, x, Val(dims))
```

Comment:
For now the default should be `dims::Union{Integer,Nothing}=nothing`, and a deprecation warning in favor of `dims=2` should be printed when `dims === nothing`. Then we'll be able to switch the default later.

Reply:
OK